document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoDocStoreTest.java

```diff
@@ -1102,4 +1106,68 @@ public void test_bulkOperationOnArrayValue_setOperation() throws Exception {
       assertEquals(expectedAttributesJsonNode, attributesJsonNode);
     }
   }
+
+  @Test
```

Comment:
The new Query API integration tests are being added in `MongoQueryExecutorIntegrationTest`.

Reply:
Added.